src/Codeception/Subscriber/ErrorHandler.php

```diff
@@ -80,7 +80,8 @@ public function errorHandler($errno, $errstr, $errfile, $errline, $context = arr
             return false;
         }
 
-        throw new \PHPUnit\Framework\Exception($errstr, $errno);
+        $relativePath = codecept_relative_path($errfile);
+        throw new \PHPUnit\Framework\Exception("$errstr at $relativePath:$errline", $errno);
```

Comment:
```suggestion
        throw new \PHPUnit\Framework\Exception("{$errstr} at {$relativePath}:{$errline}", $errno);
```

Reply:
I don't think that adding curly braces is an improvement.
I would rather do something about variable names.